client/cli/src/client_proxy.rs

```diff
@@ -329,6 +331,48 @@ impl ClientProxy {
         }
     }
 
+    /// Allow executing arbitrary script in the network.
+    pub fn allow_custom_script(
```

Comment:
Nit: `enable_custom_scripts` might be slightly clearer; `allow_custom_script` sounds like you might be allowing a single custom script.